index.Rmd

```diff
@@ -131,58 +124,57 @@ In its most basic use-case, `read_csv` expects that the data file:
+function. However, given that `read_csv` is not included in the base 
+installation of R, we will have to load it from somewhere else before we can use
+it. The place we will load it from is an R package. An R package is a collection
+of functions that once loaded, can be used in addition to the built-in R package
+functions. The `read_csv` function, in particular, can be made accessible 
+through loading the `tidyverse` package (more on this later), using the 
+`library` function.
+
+After loading the `tidyverse` package, we can call the `read_csv` function and 
+pass it a single argument: the name of the file, `"can_lang.csv"`. We have to put quotes around file names and other letters and words that we use in our code to distinguish it from the special words that make up the R programming language.  The file's name is the only argument we need to provide because our file satisfies everything else the `read_csv` function expects in the default use-case (which we just discussed). Later in the course, we'll learn more about dealing with more complicated files where the default arguments are not appropriate, for example, files that use spaces or tabs to separate the columns instead of commas.
 
 ```{r load_state_property_data, warning=FALSE, message=FALSE}
 library(tidyverse)
 read_csv("data/can_lang.csv")
 ```
-Above you can also see something neat that Jupyter does to help us understand our code: it colours text depending on its meaning in R. For example, 
-you'll note that functions get bold green text, while letters and words surrounded by quotations like filenames get blue text.
+
 
 > **In case you want to know more (optional):** 
-> We use the `read_csv` function from the `tidyverse` instead of the base R function `read.csv` because it's faster and it creates a nicer variant of the base R data frame called a *tibble*. 
+> We use the `read_csv` function from the `tidyverse` instead of the base R function `read.csv` because it's faster, and it creates a nicer variant of the base R data frame called a *tibble*. 
 > This has several benefits that we'll discuss in further detail later in the course.
 
-## Assigning value to a data frame
+## Giving a data frame a name
 
-When we loaded the language data collected in the 2016 Canadian census in R above using `read_csv`, we did not give this data frame a name, so it was 
-just printed to the screen and we cannot do anything else with it. That isn't very useful; what we would like to do is give a name to the data frame that `read_csv` outputs
-so that we can use it later for analysis and visualization. 
+When we loaded the language data collected in the 2016 Canadian census in R above using `read_csv`, we did not give this data frame a name. Therefore the data was just printed on the screen, and we cannot do anything else with it. That isn't very useful; we would like to give a name to the data frame that `read_csv` outputs to use later for analysis and visualization. 
```

Comment:
```suggestion
When we loaded the language data collected in the 2016 Canadian census in R above using `read_csv`, 
we did not give this data frame a name. Therefore the data was just printed on the screen, 
and we cannot do anything else with it. That isn't very useful. What would be more useful
would be to give a name to the data frame that `read_csv` outputs, 
so that we could refer to it later for analysis and visualization. 
```